code/controllers/subsystem/economy.dm

```diff
@@ -1,20 +1,32 @@
 #define VIP_BUDGET_BASE rand(8888888, 11111111)
+#define BUDGET_RATIO_TYPE_A 1
+#define BUDGET_RATIO_TYPE_B 2
 
 SUBSYSTEM_DEF(economy)
 	name = "Economy"
 	wait = 5 MINUTES
 	init_order = INIT_ORDER_ECONOMY
 	runlevels = RUNLEVEL_GAME
 	var/roundstart_paychecks = 5
-	var/budget_pool = 25000
-	var/list/department_accounts = list(ACCOUNT_CIV = ACCOUNT_CIV_NAME,
-										ACCOUNT_ENG = ACCOUNT_ENG_NAME,
-										ACCOUNT_SCI = ACCOUNT_SCI_NAME,
-										ACCOUNT_MED = ACCOUNT_MED_NAME,
-										ACCOUNT_SRV = ACCOUNT_SRV_NAME,
-										ACCOUNT_CAR = ACCOUNT_CAR_NAME,
-										ACCOUNT_SEC = ACCOUNT_SEC_NAME)
-	var/list/nonstation_accounts = list(ACCOUNT_VIP = ACCOUNT_VIP_NAME)
+	var/budget_pool = 25200 // "25000 / 7 = 3571" is ugly.
+	var/list/department_accounts = list(ACCOUNT_CIV_ID = ACCOUNT_CIV_NAME,
+										ACCOUNT_SRV_ID = ACCOUNT_SRV_NAME,
```

Comment:
i hate this indentation, make it equal to the above

Reply:
It is github issue. It is pretty fine in dev environment, but I will find if I can make it better